dali/pipeline/data/tensor_test.cc

```diff
@@ -368,11 +368,11 @@ TYPED_TEST(TensorTest, TestShareData) {
 }
 
 TYPED_TEST(TensorTest, TestCopyToTensorList) {
-  std::vector<Tensor<TypeParam>> tensors(16);
+  TensorVector<CPUBackend> tensors(16);
```

Comment:
Why change from parametrized backend to only CPUBackend here?
Also, as you changed the std::vector to TensorVector it would be good to test both modes of TensorVector, the contiguous (backed by TL) and noncontiguous (separate Tensors).

Reply:
Reverting to TypeParam.
Testing contiguous copy seems a bit excessive when there's no special code path for such case. When we added, it's going to be useful, though.